openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -715,6 +718,13 @@ port_INLINE bool ieee154e_processIEs(OpenQueueEntry_t* pkt, uint16_t* lenIE) {
             // at this point, ASN and frame length are known
             // the current slotoffset can be inferred
             ieee154e_syncSlotOffset();
+            schedule_syncSlotOffset(ieee154e_vars.slotOffset);
+            ieee154e_vars.nextActiveSlotOffset = schedule_getNextActiveSlotOffset();
+            /* 
+            infer the asnOffset based on the fact that
+            ieee154e_vars.freq = 11 + (asnOffset + channelOffset)%16 
+            */
+            ieee154e_vars.asnOffset = ieee154e_vars.freq - 11 - schedule_getChannelOffset();

```

Comment:
wow, this is a dependent we don't want.
What is I want to run the network on a single frequency, or change my hopping sequence?